docusaurus/docs/reference/tunnelers/linux/linux-tunnel-troubleshooting.md

```diff
@@ -23,29 +23,105 @@ journalctl -xeu ziti-edge-tunnel.service
+ziti-edge 872509 ziti  mem       REG              253,1   202904 16649478 /usr/lib/x86_64-linux-gnu/liblzma.so.5.4.1
+ziti-edge 872509 ziti  mem       REG              253,1   807936 16646759 /usr/lib/x86_64-linux-gnu/libsystemd.so.0.32.0
+ziti-edge 872509 ziti  mem       REG              253,1  2216304 16685451 /usr/lib/x86_64-linux-gnu/libc.so.6
+ziti-edge 872509 ziti  mem       REG              253,1    68552 16685824 /usr/lib/x86_64-linux-gnu/libresolv.so.2
+ziti-edge 872509 ziti  mem       REG              253,1   940560 16685551 /usr/lib/x86_64-linux-gnu/libm.so.6
+ziti-edge 872509 ziti  mem       REG              253,1   240936 16684477 /usr/lib/x86_64-linux-gnu/ld-linux-x86-64.so.2
+ziti-edge 872509 ziti    0r      CHR                1,3      0t0        5 /dev/null
+ziti-edge 872509 ziti    1u     unix 0xffff934d4455fb40      0t0  2428504 type=STREAM
+ziti-edge 872509 ziti    2u     unix 0xffff934d4455fb40      0t0  2428504 type=STREAM
+ziti-edge 872509 ziti    3u  a_inode               0,14        0     2081 [eventpoll]
+ziti-edge 872509 ziti    4r     FIFO               0,13      0t0  2428507 pipe
+ziti-edge 872509 ziti    5w     FIFO               0,13      0t0  2428507 pipe
+ziti-edge 872509 ziti    6r     FIFO               0,13      0t0  2428508 pipe
+ziti-edge 872509 ziti    7w     FIFO               0,13      0t0  2428508 pipe
+ziti-edge 872509 ziti    8u  a_inode               0,14        0     2081 [eventfd]
+ziti-edge 872509 ziti    9u      CHR             10,200     0t81      137 /dev/net/tun
+ziti-edge 872509 ziti   10u  netlink                         0t0  2435172 ROUTE
+ziti-edge 872509 ziti   11r      CHR                1,3      0t0        5 /dev/null
+ziti-edge 872509 ziti   12u     unix 0xffff934d4b2e0000      0t0  2426658 /tmp/.ziti/ziti-edge-tunnel.sock type=STREAM
+ziti-edge 872509 ziti   13u     unix 0xffff934d4b2e72c0      0t0  2426659 /tmp/.ziti/ziti-edge-tunnel-event.sock type=STREAM
+ziti-edge 872509 ziti   14u     IPv4            2431394      0t0      TCP 192.168.1.149:57706->20.253.234.163:443 (ESTABLISHED)
+ziti-edge 872509 ziti   15u     IPv4            2431392      0t0      TCP 192.168.1.149:42164->52.149.130.167:443 (ESTABLISHED)
+ziti-edge 872509 ziti   17u     IPv4            2431391      0t0      TCP 192.168.1.149:52554->20.127.103.3:443 (ESTABLISHED)
+ziti-edge 872509 ziti   18u     IPv4            2431393      0t0      TCP 192.168.1.149:57770->20.219.156.230:443 (ESTABLISHED)
+ziti-edge 872509 ziti   19u      REG               0,37      366   115033 /tmp/ziti-tunnel-routes.1xo1fA (deleted)
+```
 
 ## Process keeps crashing
 
-If the tunneller is crashing then it may be crucial to collect and analyze the core dump file. You may need to enable saving core dumps depending upon your OS configuration.
+If the tunneller is crashing, collecting and analyzing the core dump file may be crucial. You may need to enable saving core dumps depending upon your OS configuration.
```

Comment:
remove 'you' and change to factual instead of converstaional, more 'reference' style